docs/resources/api_authentication_integration_with_jwt_bearer.md

```diff
@@ -53,7 +55,6 @@ resource "snowflake_api_authentication_integration_with_jwt_bearer" "test" {
 - `oauth_access_token_validity` (Number) Specifies the default lifetime of the OAuth access token (in seconds) issued by an OAuth server.
 - `oauth_authorization_endpoint` (String) Specifies the URL for authenticating to the external service.
 - `oauth_client_auth_method` (String) Specifies that POST is used as the authentication method to the external service. If removed from the config, the resource is recreated. Valid values are (case-insensitive): `CLIENT_SECRET_POST`.
```

Comment:
what about this one? It has also just one option?

Reply:
In this case, this `property_value` from description is empty, but `property_default` is `CLIENT_SECRET_BASIC`, so I think we should leave this.

Reply:
Okay, but is there any difference in the object's behavior? If not, then I don't see any reason to keep it.

Reply:
We will follow up in SNOW-1526377